src/App.js

```diff
@@ -19,71 +27,116 @@ export default function App() {
   const StyledNav = styled.nav`
     display: grid;
     grid-auto-flow: column;
-    grid-gap: 2px;
     font-family: Helvetica, sans-serif;
-    background-color: grey;
-    font-style: grey;
-    text-decoration: none;
-    overflow: hidden;
+    background-color: rgba(187, 187, 187, 0.4);
+    position: fixed;
+    bottom: 0;
+    width: 100%;
   `
   const StyledNavLink = styled(NavLink)`
+    height: 48px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
     font-family: Helvetica, sans-serif;
     text-decoration: none;
+    border: solid white 2px;
+    color: white;
+  `
+  const StyledHeader = styled.div`
+    height: 48px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: fixed;
+    top: 0;
+    background-color: rgba(187, 187, 187, 0.4);
+    width: 100%;
+    border: solid white 2px;
+    color: white;
+    font-family: Helvetica, sans-serif;
+    margin-bottom: 10px;
+    overflow: scroll;
   `
-  const [cardData, setCardData] = useState([
-    {
-      title: 'axel',
-      _id: 'gfhdjdfhjd',
-      location: 'dfdd',
-      smell: 'Okay',
-      optic: 'Bio-Tonne',
-      category: 'Frucht',
-      comments: [
-        { name: 'toyah', message: 'lol' },
-        { name: 'Toto', message: 'egeh' },
-      ],
-    },
-  ])
 
-  // useEffect(() => {
-  //   saveCardsToStorage(cardData)
-  // }, [cardData])
+  const [cardData, setCardData] = useState(getCardsFromStorage())
```

Comment:
You might want to delete the things you commented out. 
Stichwort Clean Code :)